### PR TITLE
Update unit_dynamic_collision_volume.lua to respect sphere default

### PR DESCRIPTION
**Pull request by drivver44**
minor hitbox automation shape change to respect sphere as default instead of CYLY
this affects all units with the "canfly" tag

### CODE REVIEW

**Comment by drivver44** [comment]
do not merge at this time i have a number of changes that need to go into air units before this gets merged

**Review by Zecrus2 — APPROVED** [review]

**Pull request merged**
